portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,72 @@
+body {
+    font-family: 'Sora', Sans-serif;
+    display: flex;
+    flex-direction: column;
+}
+
+.title {
```

Comment:
nit: maybe rename it customized-title? It will be clear that it's a [CSS class selector](https://www.w3schools.com/cssref/sel_class.asp) not a [tag](https://www.w3schools.com/html/html_css.asp) applicable to all html `<title>` elements.

Even better, consider using the `h1..6` tags instead of `title` and `sub-title` (and styling them here if you prefer (e.g. same way you did for `body` tag above)). [Ref](https://www.w3schools.com/tags/tag_hn.asp)